### ISSUE TITLE
Not Working

### ISSUE THREAD

**Issue by CShehan** [Athlon1600/youtube-downloader#15]
It's working fine just one month, but now it's not work for me,

but you https://api.unblockvideos.com/youtube_downloader?id=nxmp6eb_QgI worked fine

can you give an any update?

**Comment by ToxicTurtle5** [comment]
Looks like YouTube videos with ciphers give a 403 header whereas non-ciphered videos work fine.

**Comment by CShehan** [comment]
then how to fix that?

**Comment by ToxicTurtle5** [comment]
No idea at the moment, sorry.

**Comment by scorninpc** [comment]
That problem appear only on localhost. If you host the script in some host, works. I'm working on why, but not success. Maybe sending some header thats work

**Comment by ToxicTurtle5** [comment]
I'm running on shared hosting. Getting the same issue. (Hostinger)

**Comment by scorninpc** [comment]
Strange, mine only works by hosting. I'll keep trying

**Comment by ToxicTurtle5** [comment]
This is really strange, i ran it on localhost and it works just fine...

**Comment by scorninpc** [comment]
I realized that each output ip returns a different domain. Using proxy, every output show different URL for the same video/itag. I think every output got an cluster

https://imgur.com/IR6a1sU

I leave this issue! If you try this script, y'll see you need run alot times to work just one time. It's just block from Google

`<?php

set_time_limit(0);

require "YouTubeDownloader.php";

$yt = new YouTubeDownloader();
$links = $yt->getDownloadLinks("https://www.youtube.com/watch?v=w8ja3NeUC_E");

// Get first link
$link = current($links);
var_dump($link['url']);

// Download
$fp = fopen("/PATH/teste.mp4", 'w+');                 // < ------------------- Change path
$ch = curl_init($link['url']);
curl_setopt($ch, CURLOPT_TIMEOUT, 50);
curl_setopt($ch, CURLOPT_FILE, $fp); 
curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
curl_exec($ch); 
curl_close($ch);
fclose($fp);`

I run about 8~10 times to work

**Comment by CShehan** [comment]
can you place here full code?

**Comment by ToxicTurtle5** [comment]
Seems like a quite unefficent approach but it helped alot on locating the problem. Thanks!

**Comment by ToxicTurtle5** [comment]
Magically worked again.. looks like it was a host issue.

**Comment by rayhanz** [comment]
Duplicate of #12

**Comment by scorninpc** [comment]
I dont think this is the same issue. The #12 issue is a privacy config of the videos. This is some block of youtube from host

I think this block occur when youtube find backup server. Some servers work, some others no
